rebar.config.script

```diff
@@ -1,7 +1,7 @@
 IsRebar3 = case application:get_key(rebar, vsn) of
                {ok, VSN} ->
                    [VSN1 | _] = string:tokens(VSN, "-"),
-                   [Maj, Min, Patch] = string:tokens(VSN1, "."),
+                   [Maj, Min, Patch | _] = string:tokens(VSN1, "."),
```

Comment:
this change won't handle old versions of rebar3 around. We need a case there or maybe just `[Maj | _]` ?